engine/src/main/resources/templates/ImageBuilder/launchSuccessInstantEmailBodyV2.html

```diff
@@ -0,0 +1,82 @@
+{@boolean renderSection1=true}
+{@boolean renderSection2=true}
+{@boolean renderButtonSection2=true}
+{@boolean hideSubtitleSection2=true}
+{#include Common/insightsEmailBody}
+{#content-title}
+Images - Red Hat Enterprise Linux
+{/content-title}
+{#content-title-section1}
+Instances launched successfully
+{/content-title-section1}
+{#content-title-right-part-section1}
+{action.events.size}
+{/content-title-right-part-section1}
+{#content-subtitle-section1}
+via {#switch action.context.provider}
+{#case "aws"}
+Amazon web services
+{#case "azure"}
+Microsoft Azure
+{#case "gcp"}
+Google cloud
+{/switch}, on {action.timestamp.toUtcFormat()}
```

Comment:
Timestamp should be used only if different than email timestamp, is that the case ?
If yes we can keep it here.